tests/rptest/tests/e2e_shadow_indexing_test.py

```diff
@@ -236,6 +237,67 @@ def test_write_with_node_failures(self):
         ctx.assert_actions_triggered()
 
 
+class EndToEndCloudRetentionTest(EndToEndShadowIndexingBase):
+    segment_size = EndToEndShadowIndexingBase.segment_size // 4
+    retention_bytes = 10 * segment_size
+    topics = (TopicSpec(name=EndToEndShadowIndexingBase.s3_topic_name,
+                        partition_count=1,
+                        replication_factor=3,
+                        retention_bytes=retention_bytes,
+                        segment_bytes=segment_size), )
+
+    def __init__(self, test_context):
+        super().__init__(test_context,
+                         extra_rp_conf={
+                             'default_topic_replications': self.num_brokers,
+                             'cloud_storage_housekeeping_interval_ms': 1000 * 2
+                         })
+
+    @cluster(num_nodes=4, log_allow_list=CHAOS_LOG_ALLOW_LIST)
+    def test_retention_with_node_failures(self):
```

Comment:
It might be worth using a skip_debug_mode decorator on this one: things like producing+restarts can easily run into rare failures in debug mode (debug mode is pretty slow at recovery)

Reply:
Done.